marketdata/rest.go

```diff
@@ -1754,39 +1663,29 @@ func GetCryptoSnapshots(symbols []string, req GetCryptoSnapshotRequest) (map[str
 	return DefaultClient.GetCryptoSnapshots(symbols, req)
 }
 
-// GetLatestCryptoPerpsTrade returns the latest trade for a given crypto perp symbol
-func GetLatestCryptoPerpsTrade(symbol string, req GetLatestCryptoTradeRequest) (*CryptoPerpsTrade, error) {
-	return DefaultClient.GetLatestCryptoPerpsTrade(symbol, req)
-}
-
-// GetLatestCryptoPerpsTrades returns the latest trades for the given crypto perps symbols
-func GetLatestCryptoPerpsTrades(symbols []string, req GetLatestCryptoTradeRequest) (map[string]CryptoPerpsTrade, error) {
-	return DefaultClient.GetLatestCryptoPerpsTrades(symbols, req)
-}
-
-// GetLatestCryptoPerpsQuote returns the latest quote for a given crypto perps symbol
-func GetLatestCryptoPerpsQuote(symbol string, req GetLatestCryptoQuoteRequest) (*CryptoPerpsQuote, error) {
-	return DefaultClient.GetLatestCryptoPerpsQuote(symbol, req)
+// GetLatestCryptoPerpTrade returns the latest trade for a given crypto perp symbol
+func GetLatestCryptoPerpTrade(symbol string, req GetLatestCryptoTradeRequest) (*CryptoPerpTrade, error) {
+	return DefaultClient.GetLatestCryptoPerpTrade(symbol, req)
 }
 
-// GetLatestCryptoPerpsBar returns the latest bar for a given crypto perps symbol
-func GetLatestCryptoPerpsBar(symbol string, req GetLatestCryptoBarRequest) (*CryptoPerpsBar, error) {
-	return DefaultClient.GetLatestCryptoPerpsBar(symbol, req)
+// GetLatestCryptoPerpTrades returns the latest trades for the given crypto perpetual futures
+func GetLatestCryptoPerpTrades(symbols []string, req GetLatestCryptoTradeRequest) (map[string]CryptoPerpTrade, error) {
+	return DefaultClient.GetLatestCryptoPerpTrades(symbols, req)
 }
 
-// GetLatestCryptoPerpsQuotes returns the latest quotes for the given crypto perps symbols
-func GetLatestCryptoPerpsQuotes(symbols []string, req GetLatestCryptoQuoteRequest) (map[string]CryptoPerpsQuote, error) {
-	return DefaultClient.GetLatestCryptoPerpsQuotes(symbols, req)
+// GetLatestCryptoPerpQuote returns the latest quote for a given crypto perpetual future
+func GetLatestCryptoPerpQuote(symbol string, req GetLatestCryptoQuoteRequest) (*CryptoPerpQuote, error) {
+	return DefaultClient.GetLatestCryptoPerpQuote(symbol, req)
 }
 
-// GetCryptoPerpsSnapshot returns the snapshot for a given crypto symbol
-func GetCryptoPerpsSnapshot(symbol string, req GetCryptoSnapshotRequest) (*CryptoPerpsSnapshot, error) {
-	return DefaultClient.GetCryptoPerpsSnapshot(symbol, req)
+// GetLatestCryptoPerpBar returns the latest bar for a given crypto perpetual future
```

Comment:
`GetLatestCryptoPerpBars` is missing, and please put both that and this function under `GetLatestCryptoPerpQuotes`

Reply:
Done.